go/internal/test/fakes/ac.go

```diff
@@ -66,6 +74,8 @@ func (c *ActionCache) GetWrites(d digest.Digest) int {
 
 // GetActionResult returns a stored result, if it was found.
 func (c *ActionCache) GetActionResult(ctx context.Context, req *repb.GetActionResultRequest) (res *repb.ActionResult, err error) {
+	c.mu.RLock()
```

Comment:
This updates the reads map so should be a standard Lock

Reply:
Done.